docs/tutorials/transforms.ipynb

```diff
@@ -619,60 +606,65 @@
     ")\n",
     "\n",
     "device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
-    "batch_cpu = dict(image=torch.randn(64, 13, 512, 512).to(\"cpu\"))\n",
-    "batch_gpu = dict(image=torch.randn(64, 13, 512, 512).to(device))\n",
     "\n",
-    "transforms_gpu = transforms.to(device)"
-   ],
-   "execution_count": 10,
-   "outputs": []
+    "\n",
+    "def get_batch_cpu():\n",
+    "    return dict(image=torch.randn(64, 13, 512, 512).to(\"cpu\"))\n",
+    "\n",
+    "\n",
+    "def get_batch_gpu():\n",
+    "    return dict(image=torch.randn(64, 13, 512, 512).to(device))\n",
+    "\n",
+    "\n",
+    "transforms_gpu = deepcopy(transforms).to(device)"
+   ]
   },
   {
    "cell_type": "code",
+   "execution_count": 14,
    "metadata": {
     "colab": {
      "base_uri": "https://localhost:8080/"
     },
     "id": "vo43CqJ4IIXE",
     "outputId": "d5de6f59-6705-45a5-94e0-eb31cf1f703b"
    },
-   "source": [
-    "%%timeit -n 1 -r 1\n",
-    "_ = transforms(batch_cpu)"
-   ],
-   "execution_count": 11,
    "outputs": [
     {
-     "output_type": "stream",
      "name": "stdout",
+     "output_type": "stream",
      "text": [
-      "1 loop, best of 1: 12.3 s per loop\n"
+      "2.89 s ± 65.8 ms per loop (mean ± std. dev. of 5 runs, 1 loop each)\n"
      ]
     }
+   ],
+   "source": [
+    "%%timeit -n 1 -r 5\n",
+    "_ = transforms(get_batch_cpu())"
```

Comment:
```suggestion
    "transforms(get_batch_cpu())"
```

Reply:
I think this will end up printing the transformed tensor which is not what we want right?

Reply:
Oh true